src/models/Token/index.js

```diff
@@ -1,17 +1,53 @@
 // @flow
-import createToken from './createToken'
-import retrieveToken from './retrieveToken'
-import validateToken from './validateToken'
-export * from './createToken'
-
-export type Token = {
-  id: string,
-  token: string,
-  userId: string
-}
+import { Model } from 'objection'
+import User from '@models/User'
+
+class Token extends Model {
+  $beforeInsert() {
+    this.createdAt = new Date().toISOString()
+  }
+
+  $beforeUpdate() {
+    this.updatedAt = new Date().toISOString()
+  }
+
+  static get tableName(): string {
+    return 'tokens'
+  }
+
+  static get idColumn(): string {
+    return 'id'
+  }
 
-export default {
-  createToken,
-  retrieveToken,
-  validateToken
+  static updatedAt: Date
+  static createdAt: Date
+  static token: string
```

Comment:
☝️ I don’t think those need to be static properties, even for Flow. You also didn’t do that for the Project class.